calico_kubernetes/calico_kubernetes_cni.py

```diff
@@ -28,14 +31,21 @@
 from logutils import configure_logger
 
 ETCD_AUTHORITY_ENV = 'ETCD_AUTHORITY'
+CALICOCTL_PATH = os.environ.get('CALICOCTL_PATH', '/usr/bin/calicoctl')
```

Comment:
The default value here should be "calicoctl" rather than "/usr/bin/calicoctl"

This means that sh.Command will search the current PATH for calicoctl.